data/ExperimentDesignPresets.ts

```diff
@@ -0,0 +1,26 @@
+import {AAExperiment} from "../types/experiments";
+
+interface Preset<T> {
+  name: string;
+  description: string;
+  presets: Partial<T>
+}
+
+export const AAType: Preset<AAExperiment> = {
+  name: "A/A Experiment",
+  description: "A design for diagnostic testing of targeting or enrollment. Fixed to 1% of the population.",
```

Comment:
Is this fixed to 1% as a strawman thing to show off that it's possible, or is that a real proposal?

Reply:
Not real – what do you think the real values should be?